### DISCUSSION TITLE
Community-driven wiki

### DISCUSSION THREAD

**Discussion by MaelImhof** [logisim-evolution/logisim-evolution#1413] [Ideas]
Hello!

I'm part of a group of several students that use *logisim-evolution* for our lessons. Thanks a lot for all of your work, the software is great!

However, some students encounter troubles when trying to learn how to use logisim-evolution, and I need to say it's hard to find good and simple documentation, that's to say quick problem fix without having an entire PDF to read.

The wiki is currently not much used and not complete in my opinion. Some of my classmates and I would be happy to create new pages and helping new users to dive in.
In order to do that, **we need permissions to manage the wiki**. We're currently only able to read, but it is possible to [enable community-driven wikis](https://docs.github.com/en/communities/documenting-your-project-with-wikis/changing-access-permissions-for-wikis), if only you wish to.

Please let us know if you do not want to open the accesses, we'll create our own documentation, that will still be accessible by anyone.

**Comment by MarcinOrlowski** [comment]
We'd love to see more works on the docs so you are welcome to contribute! Unfortunately project documentation is currently part of the main repository so if you want to update it, you need to go via regular fork->change->pull-request cycle. We had opened ticket [Move user documentation to separate repo (#824)](https://github.com/logisim-evolution/logisim-evolution/issues/824) to discuss separating docs from the code base, but this is still not done nor really decided how to do it correctly. You are welcome to join the debate in the above ticket to push things forward.

I temporarily disabled wiki pages as in its current form it is pretty much useless for general audience, but (@BFH-ktt1, @maehne) please re-enable if you think this is incorrect or I  missed any reason it should stay available. Unfortunately Github's wiki does not support normal git driven contributions so it's most likely useless from our perspective anyway.

**Reply by MaelImhof** [reply]
Oh yeah you got a point, maybe creating a Github pages new repository would be a better way. I'll join the ticket! Thanks!

**Comment by BFH-ktt1** [comment]
I re-enabled the wiki, with public access, so please feel free to contribute to it!

**Reply by MaelImhof** [reply]
Thanks!
